src/sentry/ownership/grammar.py

```diff
@@ -141,8 +141,8 @@ def test_with_munged(
         self, data: PathSearchable, munged_data: tuple[Sequence[Mapping[str, Any]], Sequence[str]]
```

Comment:
I think this can this just fully replace `test` now?

Reply:
yes, need to rewrite a bunch of tests. can update this PR to do so

Reply:
👍 definitely doesn't need to happen with this PR specifically in my opinion if you want to get this out sooner